aspnet/tutorials/first-mvc-app/index.rst

```diff
@@ -1,5 +1,5 @@
-Building your first ASP.NET Core MVC application
-------------------------------------------------
+Building your first MVC 6 application
```

Comment:
MVC 6 is now ASP.NET Core MVC. The original name was correct, except for the lack of title capitalization: "Building Your First ASP.NET Core Application"